modules/installation-vsphere-machines.adoc

```diff
@@ -62,7 +62,7 @@ endif::openshift-origin[]
 .. Click the *VMs and Templates* view.
 .. Right-click the name of your datacenter.
 .. Click *New Folder* -> *New VM and Template Folder*.
-.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, create a folder with the same name as the infrastructure ID.
+.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, then create a folder with the same name as the infrastructure ID. You use this folder name so vCenter dynamically provisions storage in the appropriate location for its Workspace configuration.
```

Comment:
Do you need the _then_ here? _If you did this, do that_ seems fine to me.

Reply:
s/so/so that ??

A quick internet search reveals: “So that” tells us for what purpose something is done. What was the intention or plan? In contrast, “so” tells us the effect. It tells us what happened as a result. It's probably a toss-up in this case. But so that reads better to me. Up to you.